cmd/utils/tss_options.go

```diff
@@ -116,3 +116,47 @@ func ErrorHandlerServiceNonJitterChannelMaxRetriesOption(configKey *int) *config
 		Required:    true,
 	}
 }
+
+func WebhookHandlerServiceChannelMaxBufferSizeOption(configKey *int) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:        "webhook-service-channel-max-buffer-size",
+		Usage:       "Set the buffer size of the webhook serive channel.",
+		OptType:     types.Int,
+		ConfigKey:   configKey,
+		FlagDefault: 100,
+		Required:    true,
+	}
+}
+
+func WebhookHandlerServiceChannelMaxWorkersOptions(configKey *int) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:        "webhook-service-channel-max-workers",
+		Usage:       "Set the max number of workers for the webhook serive channel.",
+		OptType:     types.Int,
+		ConfigKey:   configKey,
+		FlagDefault: 10,
+		Required:    true,
+	}
+}
+
+func WebhookHandlerServiceChannelMaxRetriesOption(configKey *int) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:        "webhook-service-channel-max-retries",
+		Usage:       "Set the max number of times to ping a webhook before quitting.",
+		OptType:     types.Int,
+		ConfigKey:   configKey,
+		FlagDefault: 3,
+		Required:    true,
+	}
+}
+
+func WebhookHandlerServiceChannelMinWaitBtwnRetriesMSOption(configKey *int) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:        "webhook-service-channel-min-wait-between-retries",
+		Usage:       "The minumum amout of time to wait before repining the webhook url",
```

Comment:
Is this in `seconds`, right?
```suggestion
		Usage:       "The minimum amount of time to wait before resending the request to the webhook url",
```

Reply:
Corrected this typo. As for wait times, typically from what I've seen in exponential backoff implementations is a wait time in the order of milliseconds so I'm doing the same here